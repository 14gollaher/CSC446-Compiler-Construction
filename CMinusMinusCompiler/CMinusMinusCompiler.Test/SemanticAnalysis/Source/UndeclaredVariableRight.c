int main(int x)
{
	/*x = y;*/
	return 100;
}

int foo()
{
	int y;
	y = x;
	return y;
}
