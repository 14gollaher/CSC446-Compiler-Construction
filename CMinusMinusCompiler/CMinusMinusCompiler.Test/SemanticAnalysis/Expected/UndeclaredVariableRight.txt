** Symbol Table at Depth 2 **
Lexeme          Class        Depth     Offset    Type         Size      Value(r)
--------------------------------------------------------------------------------
x               Variable     2         0         IntToken     2         -


ERROR: Line 10 Use of undeclared variable x
Completed processing UndeclaredVariableRight.c
